hermes/bin/src/runtime_extensions/wasi/clocks/monotonic/host.rs

```diff
@@ -11,12 +12,12 @@ impl Host for HermesState {
     /// The clock is monotonic, therefore calling this function repeatedly will
     /// produce a sequence of non-decreasing values.
     fn now(&mut self) -> wasmtime::Result<Instant> {
-        todo!()
+        now_impl()
```

Comment:
I would use this: https://doc.rust-lang.org/std/time/struct.Instant.html

At start of world, get a GLOBAL `base` "instant" now().
Then for every call, just return `base.elapsed().as_nanos()`